extensions/amp-accordion/1.0/README.md

```diff
@@ -636,25 +611,21 @@ Or via `className`:
 
 ##### animate
 
-If true, then uses "roll-down" / "roll-up" animation during the expansion and collapse of each section
-Default: `false`
+If true, then uses "roll-down" / "roll-up" animation during the expansion and collapse of each section Default: `false`
 
 ##### expandSingleSection
 
-If true, then expanding 1 section will automatically collapse all other sections:
-Default: `false`
+If true, then expanding 1 section will automatically collapse all other sections: Default: `false`
```

Comment:
Perhaps we want a newline before `Default: false`? Probably a better approach would to say `The default value is false`. Same for animate and expanded attributes below.